yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -236,7 +236,7 @@ describe('e2e_cross_chain_messaging', () => {
     await expect(
       l2Bridge
         .withWallet(user2Wallet)
-        .methods.claim_public(ownerAddress, bridgeAmount, ethAccount, messageKey, secretForL2MessageConsumption)
+        .methods.claim_public(bridgeAmount, ownerAddress, ethAccount, messageKey, secretForL2MessageConsumption)
```

Comment:
Why the swap here between (to, amount) -> (amount, to)? 

The erc20 standard is (to,amount) so this is likely what developers are used to

Reply:
Currently, this is the only test where we do `(to, amount)`. Everywhere else we do `(amount, to)`. That said I agree with you - should use the erc20 standard. 

Might be a bigger PR so might want to do it separately!